src/Task1/Person.java

```diff
@@ -0,0 +1,69 @@
+package Task1;
```

Comment:
I didn't explain this. Package names follow the naming convention: {domain}.{organisation}.{project}.{...}
i.e. org.AsjaHas.qamp.zadaci.zadaca5